crates/services/p2p/src/p2p_service.rs

```diff
@@ -1275,7 +1287,7 @@ mod tests {
             tokio::select! {
                 message_sent = rx_test_end.recv() => {
                     // we received a signal to end the test
-                    assert_eq!(message_sent, Some(true), "Received wrong block height!");
+                    assert_eq!(message_sent, Some(true), "Receuved incorrect or missing missing messsage");
```

Comment:
[nit]
```suggestion
                    assert_eq!(message_sent, Some(true), "Received incorrect or missing missing messsage");
```

Assertion seems to be failing here

Reply:
there is something wrong with `Request::Transactions` test others are passing, if no one solves it by tomorrow I will try it.

It seems the request doesn't even go through, the node gets disconnected every time I re-ran it with Transactions, other requests pass which is (slightly) weird